integration_tests/src/main/python/date_time_test.py

```diff
@@ -477,6 +477,24 @@ def test_formats_for_legacy_mode(format):
         {'spark.sql.legacy.timeParserPolicy': 'LEGACY',
          'spark.rapids.sql.incompatibleDateFormats.enabled': True})
 
+# mm: minute; MM: month
+@pytest.mark.skipif(not is_supported_time_zone(), reason="not all time zones are supported now, refer to https://github.com/NVIDIA/spark-rapids/issues/6839, please update after all time zones are supported")
```

Comment:
Why are we skipping instead of marking them as xfail? skip means that it will not run, xfail indicates that we expect it to fail, but it might still pass so lets at least try and run it.